### ISSUE TITLE
[v1.0.0b] - Unable to replace image

### ISSUE THREAD

**Issue by epheterson** [mealie-recipes/mealie#1404]
### First Check

- [X] This is not a feature request
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.

### What is the issue you are experiencing?

I imported a recipe URL which had a logo image instead of a dish. I then tried to replace the image with another one I found online, and although it shows the new image in the Edit view, it never sticks.

Reproduction Steps:
1. Import recipe: https://livivafoods.com/pages/shirataki-fettuccine-with-lemon-chickenand-spinach
2. Edit > Image > Enter URL: https://cdn.shopify.com/s/files/1/0056/0886/5856/articles/Low-Carb_Herbed_Spinach_Ricotta_Fettuccine_with_Lemon_Butter_Chicken_with_Mushrooms_2048x.progressive.jpg?v=1641516649
3. Save
4. Reload page

### Deployment

Docker (Synology)

### Deployment Details

```
**Details**
Version: v1.0.0beta-3
Build: 56eb0bca7142715984c8028c4c881f086e979564
Application Mode: Production
Demo Status: Not Demo
API Port: 9000
API Docs: Enabled
Database Type: sqlite
Recipe Scraper Version: 14.3.0

**Checks**
Secure Site: Yes
Server Side Base URL: Yes
LDAP Ready: No
Email Configured: No
Docker Volumes: success
```

**Comment by epheterson** [comment]
Actually, in browse views I see the correct image, though the recipe page itself always shows the original image.

**Comment by epheterson** [comment]
I tried loading in another browser, and it does appear correctly, so this may be more a browser caching issue which I'm not sure Mealie can control.

Original issue discovered in Safari, when I tried in Firefox I see the correct image in all views.

**Comment by hay-kot** [comment]
I was able to reproduce your issue, and have verified that the fix in #1427 has fixed it (at least on the demo site). Closing for now, if you're still experiencing an issue feel free to reopen.
